target/linux/ramips/dts/mt7620a_tplink_re210-v1.dts

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a_tplink_re2x0-v1.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,re210-v1", "ralink,mt7620a-soc";
+	model = "TP-Link RE210 v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "re210-v1:green:power";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi1 {
+			label = "re210-v1:green:rssi";
+			gpios = <&gpio1 1 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi2 {
+			label = "re210-v1:orange:rssi";
+			gpios = <&gpio1 2 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Optional:
Consider to rename those to rssi_low and rssi_high or something else being more close to how they are used.
I'd typically expect the orange LED to be "low", so having this called rssi2 is somewhat unexpected to me.
But since they are not used at the moment anyway, that's purely cosmetic.

Reply:
done. changed them to the suggested names and also used "red" for rssi_low, as thats the correct color of the led.